code_challenges/code_wars/football_cards.py

```diff
@@ -0,0 +1,55 @@
+def football_cards(team_cards):
+    a_players = 11
+    b_players = 11
+    a_cards = {}
+    b_cards = {}
+    while a_players > 6 and b_players > 6:
+        for card in team_cards:
+            if a_players == 6 or b_players == 6:
+                break
+            if card[0] == "A":
+                if len(card) == 3:
+                    player_number = card[1]
+                    a_cards[player_number] = a_cards.get(player_number, "") + card[2]
+                    if (
+                        a_cards[player_number] == "R"
+                        or a_cards[player_number] == "YY"
+                        or a_cards[player_number] == "YR"
+                    ):
+                        a_players -= 1
+                else:
+                    player_number = card[1] + card[2]
+                    a_cards[player_number] = a_cards.get(player_number, "") + card[3]
+                    if (
+                        a_cards[player_number] == "R"
+                        or a_cards[player_number] == "YY"
+                        or a_cards[player_number] == "YR"
+                    ):
+                        a_players -= 1
```

Comment:
Great job!  Getting it working is always the first goal.  Now lets think about how to make it a little better.

The logic above in the if block is exactly duplicated below in the else block except with different variable names.  Can you think of a way to [DRY (don't repeat yourself)](https://en.wikipedia.org/wiki/Don't_repeat_yourself) this code up some?

Consider your variables:
```python
a_players = 11
b_players = 11
a_cards = {}
b_cards = {}
```
How could your code change if you were to use a structure like:
```python
teams = {
    "A": {"players": 11, "cards": {}},
    "B": {"players": 11, "cards": {}},
}
```